### DISCUSSION TITLE
Walkthrough on how to install and setup a working instance with Slack?

### DISCUSSION THREAD

**Discussion by jwl** [Netflix/dispatch-docker#120] [Ideas]
Hi, I've been trying to get a simple instance of Dispatch running on my local machine and integrated with a small Slack environment. The documentation is thorough in outlining what the options are but it's not super clear how to get something up and running from scratch. I've pieced together something that can talk to Slack and accept commands and list incidents, but I'm running into major issues such as Dispatch being seemingly unable to create/invite users to incident channels; the WebUI is hanging when creating new incidents and granting Admin roles to a user prevents them from logging in again after logging out. 

There are also other stumbling blocks on initial setup such as leaving out that Slack plugins must be manually and individually enabled in order to get Slack to work. 

Does there exist a guide somewhere on how to get Dispatch up and running with a few essential plugins such as Slack such that the core functionality can be tested and demonstrated by the user? If not, may I suggest that the creation of such documentation be a priority?

**Comment by mvilanova** [comment]
@jwl we don't have a guide like the one you're describing at the moment, but if you have the cycles to start one, we could tag team on it.

> I'm running into major issues such as Dispatch being seemingly unable to create/invite users to incident channels; the WebUI is hanging when creating new incidents and granting Admin roles to a user prevents them from logging in again after logging out.

Would you mind opening an issue and providing more details about it?
